tests/utils/focus/getActiveElement.ts

```diff
@@ -0,0 +1,32 @@
+import '../../_helpers/shadow-input'
+import {setup} from '#testHelpers'
+import {getActiveElement, getActiveElementOrBody} from '#src/utils'
+
+describe('focus tests with not shadow root', () => {
+  test('focus input element', async () => {
+    const {element} = setup('<input />')
+
+    expect(getActiveElement(document)).toBe(element)
+  })
+
+  test('focus should be body', async () => {
+    setup('<input />', {focus: false})
+
+    expect(getActiveElementOrBody(document)).toBe(document.body)
+  })
+})
+
+describe('on shadow DOM', () => {
+  test('focus contained input element', async () => {
+    const {element} = setup('<shadow-input></shadow-input>')
+
+    expect(getActiveElement(document)).toBe(
+      element.shadowRoot?.querySelector('input'),
+    )
+  })
+  test('focus just body', async () => {
+    setup('<shadow-input></shadow-input>', {focus: false})
+
+    expect(getActiveElementOrBody(document)).toBe(document.body)
+  })
```

Comment:
This is an anti-pattern. This tests asserts that our test setup is doing nothing while suggesting that has something to do with util being used here.

Reply:
I tried making it more explicit.